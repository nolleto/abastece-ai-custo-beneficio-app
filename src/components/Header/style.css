.header h1 {
  margin-bottom: 5px;
}

.header h3 {
  margin-top: 0;
}

.logo {
  width: auto;
  height: auto;
  max-width: 80px;
  pointer-events: none;
}
