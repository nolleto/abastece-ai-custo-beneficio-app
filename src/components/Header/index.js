import React from 'react';
import './style.css';

const Header = () => (
  <header className='header'>
    <h1>Abastece Aí</h1>
    <h3>Custo Benefício</h3>
  </header>
);

export default Header;
