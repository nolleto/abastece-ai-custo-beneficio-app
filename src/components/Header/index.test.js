import React from 'react';
import Header from './index';
import { shallow } from 'enzyme';

describe('Header component', () => {
  it('matches snapshot', () => {
    const snapshot = shallow(<Header />);

    expect(snapshot).toMatchSnapshot();
  });
});
