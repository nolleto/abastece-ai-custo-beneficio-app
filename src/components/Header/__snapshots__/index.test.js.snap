// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component matches snapshot 1`] = `
<header
  className="header"
>
  <h1>
    Abastece Aí
  </h1>
  <h3>
    Custo Benefício
  </h3>
</header>
`;
